atmo/clusters/jobs.py

```diff
@@ -72,9 +72,11 @@ def update_clusters_info():
     """
     # only update the cluster info for clusters that are pending
     active_clusters = Cluster.objects.active()
+    # get the datetime of the oldest active cluster, set to the start of the day
```

Comment:
Why do you need it set to the start of the day instead of the earliest active cluster start_date?